### PR TITLE
Completed css-brochure-form challenge.

### PR DESCRIPTION
**Pull request by leanne-vu**
Screen Shot 2022-10-12 at 9 53 38 PM

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+}
+
+button {
+  background-color: #ec5d58;
+  color: white;
+  border: none;
+  border-radius: 18px;
+  padding: 0.4em;
+  width: 5.6em;
+  text-align: center;
```
Comment (Zhiwutian) [inline css-brochure-form/styles.css:55]: I would use text align on the parent of the button so you can use it to move it to the right of the div. That would eliminate the need for a lot of margin.

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Leanne, just a couple updates:
1. Add just a little more vertical space between the inputs
2.  adjust the CSS for the button as noted in the comment. 

Great work so far!

File: css-brochure-form/styles.css (hunk 1 above)
Comment (leanne-vu) [reply css-brochure-form/styles.css:66]: Screen Shot 2022-10-13 at 10 12 10 AM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**